packages/svelte/src/utils.js

```diff
@@ -107,7 +107,7 @@ export function is_capture_event(name) {
 }
 
 /** List of Element events that will be delegated */
-export const DELEGATED_EVENTS = [
+const DELEGATED_EVENTS = [
```

Comment:
This is odd, why aren't we re-using this everywhere? Surely we don't have two of them in the codebase?

Reply:
it's used by the `is_delegated` function, which _is_ used in multiple places